index.js

```diff
@@ -76,33 +76,52 @@ async function createRoom() {
 async function createRoomAndStart() {
   const createAndStartButton = document.getElementById('create-and-start');
   const copyUrl = document.getElementById('copy-url');
+  const errorTitle = document.getElementById('error-title');
+  const errorDescription = document.getElementById('error-description');
 
   createAndStartButton.innerHTML = 'Loading...';
 
   room = await createRoom();
+  if (!room) {
+    errorTitle.innerHTML = 'Error creating room';
+    errorDescription.innerHTML =
+      "If you're developing locally, please doublecheck the README instructions.";
```

Comment:
I think double-check is hyphenated. Just "check" would work too 🤷‍♂️ Whatever you think

Reply:
Thank you! "check" it is.